src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+public class DukeException extends RuntimeException {
+    /**
+     * Constructor for a new instance of DukeException.
+     *
+     * @param e The error message of the exception.
+     */
+    public DukeException(String e) {
+        super(e);
+    }
+}
+
+class UnknownCommandException extends DukeException {
```

Comment:
I believe that UnknownCommandException class should be defined in a separate file...... 🤔